pkg/virt-handler/device-manager/usb_device.go

```diff
@@ -559,6 +559,9 @@ func (l *LocalDevices) fetch(selectors []v1.USBSelector) ([]*USBDevice, bool) {
 func discoverPluggedUSBDevices() *LocalDevices {
 	usbDevices := make(map[int][]*USBDevice, 0)
 	err := filepath.Walk(PathToUSBDevices, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
```

Comment:
Does it make sense to open a separate PR to fix this?

Reply:
I can raise a separate PR, though i think other arches always have the directory, so it does not error there.

Reply:
Wll, ff the function can raise an error then it is correct to make the check. So, I would do the fix in a separate PR. Do you mind to separate the change.

Reply:
I have created https://github.com/kubevirt/kubevirt/pull/11047. I will rebase this PR as soon as it is merged.

Reply:
I have rebased this PR to pick up #11047